simulator/src/test/java/com/hedera/block/simulator/mode/ConsumerModeHandlerTest.java

```diff
@@ -17,24 +17,71 @@
 package com.hedera.block.simulator.mode;
 
 import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.*;
```

Comment:
Consider to remove this kind of wildcard imports.

Reply:
If you're using IntelliJ, you can change the default behavior to only use wildcard imports when the number of classes exceeds 999. That's how I was able to "turn off" wildcard imports:
<img width="988" alt="image" src="https://github.com/user-attachments/assets/3154fb97-2db3-4370-9ee8-1facd36bc4c7">

Reply:
Thanks, I missed this one. fixed ✅